halo2_frontend/src/plonk/lookup.rs

```diff
@@ -1,14 +1,14 @@
-use super::circuit::Expression;
+use crate::plonk::Expression;
 use halo2_middleware::ff::Field;
 use std::fmt::{self, Debug};
 
 /// Expressions involved in a lookup argument, with a name as metadata.
 /// TODO: possible to move to "halo2_backend", if moved, pub(crate) fields.
```

Comment:
I think we can remove this TODO comments. @ed255

Reply:
that's right, thanks!  Done in https://github.com/privacy-scaling-explorations/halo2/pull/290/commits/7b44b19579a97c9e2ed059098e4334a8afc64dab